src/tzstats/tzstats_reward_api.py

```diff
@@ -40,12 +41,17 @@ def get_rewards_for_cycle_map(self, cycle, rewards_type):
             self.blocks_per_cycle * self.consensus_committee_size
         )
         potential_endorsement_rewards = (
```

Comment:
@nicolasochem can you send me a resource regarding the calculation of the potential_endorsement_rewards? I am trying to match the tzkt futureRewards with this calculation. Currently they are not matching.

Reply:
@vkresch it's diligently :wink: documented on the tezos dev slack.

https://tezos-dev.slack.com/archives/CV5NX7F2L/p1649433246273169?thread_ts=1648854391.875409&cid=CV5NX7F2L

Drop me your email if you don't have access.

Reply:
I don't have access :(
Mail: viktor.kreschenski@gmail.com

Reply:
I have invited you

Reply:
@nicolasochem I compared the tzkt and tzstats potentialRewards and they seem to not match. Am I missing something? Check[ this slack message](https://tezos-dev.slack.com/archives/CV5NX7F2L/p1670716905787589?thread_ts=1648854391.875409&cid=CV5NX7F2L) for more details.

Reply:
For now I added an approximation for the assert.